### PR TITLE
v0.2.0

### PR DESCRIPTION
**Pull request by KirillSergeevich**

### CODE REVIEW

File: .github/workflows/analyze.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - uses: subosito/flutter-action@v2
+        with: 
+          flutter-version: '3.x'
+          channel: 'stable'
+
+      - name: Analyzing
+        run: flutter analyze
+
+      - name: Pana check
+        run: ./scripts/pana_check.sh
```
Comment (KirillSergeevich) [inline .github/workflows/analyze.yml:21]: @damir-abdulin new line in the end of file

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 0.2.0
+Update package to Flutter 3.10.
+Dart 3 support
+Style fixes
+
```
Comment (KirillSergeevich) [inline CHANGELOG.md:5]: please add `Bump Android Kaltura version to 4.28.3` @damir-abdulin

File: analysis_options.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  errors:
+    close_sinks: error
+    missing_required_param: error
+    missing_return: error
+    # remove line below to enable doc lints
+    public_member_api_docs: ignore
+    sort_pub_dependencies: ignore
+    unawaited_futures: ignore
+    sort_constructors_first: ignore
+    mixin_inherits_from_not_object: ignore
```
Comment (KirillSergeevich) [inline analysis_options.yaml:21]: we need almost all these rules @damir-abdulin

File: analysis_options.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+  exclude:
+    - "build/**"
+    - "**/*.g.dart"
+    - "**/*.gr.dart"
+    - "**/*.freezed.dart"
+    - "**/*.config.dart"
```
Comment (KirillSergeevich) [inline analysis_options.yaml:28]: We don't have these files, no need to ignore them @damir-abdulin

File: analysis_options.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+          - test/**
+  metrcis:
+    - cyclomatic-complexity: 10
+    - maximum-nesting-level: 5
+    - source-lines-of-code: 50
+
+
+linter:
+  rules:
+    library_private_types_in_public_api: false
```
Comment (KirillSergeevich) [inline analysis_options.yaml:70]: it's better to fix it and add a new line

File: pubspec.yaml [hunk 6]
```diff
@@ -1,10 +1,10 @@
 name: vr_player
 description: The `VrPlayer` lets you play 360° and VR videos smoothly on Android and iOS platforms
-version: 0.1.5
+version: 0.2.0
 homepage: "https://github.com/What-the-Flutter/VR-Player"
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <4.0.0"
```
Comment (KirillSergeevich) [inline pubspec.yaml:7]: let's migrate to sdk: ">=3.0.0 <4.0.0" and update all dependencies here

File: analysis_options.yaml [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    - "**/*.config.dart"
+
+dart_code_metrics:
+  extends:
+    - package:dart_code_metrics_presets/all.yaml
+  rules:
+    - prefer-moving-to-variable:
+        allowed-duplicated-chains: 3
+    - arguments-ordering: false
+    - avoid-banned-imports: false
```
Comment (KirillSergeevich) [inline analysis_options.yaml:37]: looks like it's better to delete the majority of these rules

File: .github/workflows/analyze.yml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+      - uses: actions/checkout@v3
+
+      - uses: subosito/flutter-action@v2
+        with: 
+          flutter-version: '3.x'
+          channel: 'stable'
+
+      - name: Analyzing
+        run: flutter analyze
+
```
Comment (KirillSergeevich) [inline .github/workflows/analyze.yml:19]: @damir-abdulin looks like you forgot to add DCM check

**Review by ivan-wtf — APPROVED** [review]

**Pull request merged**